src/db_options.rs

```diff
@@ -1098,6 +1098,17 @@ impl Options {
             ffi::rocksdb_options_set_wal_dir(self.inner, p.as_ptr());
         }
     }
+
+    // If true, then DB::Open() will not update the statistics used to optimize
```

Comment:
@romanz isn't it better to use doc comments here, is it ?

Reply:
@aleksuss Oops, you're right. Will fix and re-push.